terraform/a2ha-terraform/destroy/aws/main.tf

```
@@ -58,7 +53,7 @@ module "aws" {
   lb_access_logs                  = var.lb_access_logs
   tags                            = var.aws_tags
   aws_instance_profile_name       = var.backup_config_s3 == "true" ? module.s3[0].instance_profile_name : null
-  json_data                          = local.json_data
+  json_data                       = module.aws_metadata.private_ip
```

```suggestion
  json_data                       = module.aws_metadata.private_ip
  depends_on =  [module.aws_metadata.private_ip]
```
See if the above change is required if there were ever a case where aws runs before aws_metadata.

---

Done. Didn't get any issue as of that, but to make sure aws_metadata runs before aws I have added depends on.